yt_dlp/extractor/rozhlas.py

```diff
@@ -219,3 +244,133 @@ def _real_extract(self, url):
             'title': traverse_obj(data, ('series', 'title')),
             'entries': map(self._extract_video, data['playlist']),
         }
+
+
+class MujRozhlasIE(RozhlasBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?mujrozhlas\.cz/(?:[^/]+/)*(?P<id>[^/?#&]+)'
+    _TESTS = [{
+        'url': 'https://www.mujrozhlas.cz/vykopavky/ach-jo-zase-teleci-rizek-je-mnohem-min-cesky-nez-jsme-si-mysleli',
+        'info_dict': {'id': '10739193',
```

Comment:
This test fails. No `ext` field. Also, this formatting is wrong. Use the info_dict that the test_download script outputs